game/client_core.cpp

```diff
@@ -37,10 +42,16 @@ void ClientCore::run()
 
 void ClientCore::send_input()
 {
-    InputPacket packet;
+    InputPacket packet;/*
     packet.events.push_back(0);
-    packet.events.push_back(1);
-    packet.cam_angle = 2.0f;
+    packet.events.push_back(1);*/
+    packet.cam_angle = 0.0f;
+    // TODO: Get events and push it into packet
+    std::vector<int> tmp = Window::get_input_actions();
```

Comment:
Does this still leave gathering input to the server? If so I think that's fine for now just something to keep in mind